specs/container/readme.md

```diff
@@ -38,4 +38,59 @@ type ContainerModule interface {
     // Ps displays the processes running inside a container
     Ps(ID string) (error)
 }
-```
\ No newline at end of file
+```
+
+## OCI bundle
+
+The oci bundle must define `rootfs` and a `config.json`. currently flist defines only the `rootfs` of a container. There are no attached meta to define entrypoint, env variables, exposed ports, etc ...
+
+Since the flist is a tar that container only one file (the rootfs db), it's okay we add more files to the tar. I suggest we add custom meta file that contains
+the missing pieces of the bundle that we need to construct the final `config.json` file.
+
+### Flist meta
+
+Part of the `config.json` entries that can be defined byt the user during the flist build stage or when creating a container:
+
+- process.terminal
+- process.user
+- process.args
+- process.env
+- process.cwd
+- hostname
+- mounts (for persistent storage)
+
+Next to these field we will also authorized:
+
+- exposed ports (this will allow auto port forward to the application ports automatically if asked)
+
+### Runtime meta
```

Comment:
I can see u dropped the pre-defined profiles idea from the specs (privileged and unprivileged), any reason why ?

Reply:
I'm not discarding predefined profile, but just privileged and unprivileged is not right. There won't be such a thing a privileged container. All container must be as secure as possible. Maybe some profile can allow certain things for certain workloads. But then it is going to be very limited capabilities used with certain chosen flist.